<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="boton_volver">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>500</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#boton_volver{
    background-color:qlineargradient(spread:pad, x1:0.198, y1:0.903, x2:1, y2:0, stop:0.354167 rgba(213, 213, 213, 255), stop:0.661458 rgba(147, 147, 147, 255));
    color: rgba(0, 0, 0, 189);
    border-radius: 5px;
    font-family: &quot;Franklin Gothic Heavy&quot;;
    font-size: 14pt;
}

QPushButton#boton_volver:hover {
    background-color: qlineargradient(spread:pad, x1:0.198, y1:0.903, x2:1, y2:0, stop:0.354167 rgba(233, 233, 213, 255), stop:0.661458 rgba(147, 147, 147, 255));
    color: rgba(255, 255, 255, 210);
    border-radius: 5px;
    font-family: &quot;Franklin Gothic Heavy&quot;;
}

QPushButton#boton_volver:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgba(105, 118, 132, 200);
    font-family: &quot;Franklin Gothic Heavy&quot;;
}
</string>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_transversal">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>201</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>Corte Transversal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_coronal">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Corte Coronal</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_sagital">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>520</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#slider_sagital::groove:horizontal {
    border: 1px solid #999999;
    height: 8px;
    background: qlineargradient(
        spread:pad,
        x1:0.198, y1:0.903,
        x2:1, y2:0,
        stop:0.354167 rgba(213, 213, 213, 255),
        stop:0.661458 rgba(147, 147, 147, 255)
    );
    border-radius: 5px;
}

#slider_sagital::handle:horizontal {
    background: rgba(0, 0, 0, 189);
    border: 1px solid #5c5c5c;
    width: 18px;
    height: 18px;
    margin: -5px 0;
    border-radius: 9px;
}

#slider_sagital::sub-page:horizontal {
    background: rgba(147, 147, 147, 150);
    border-radius: 5px;
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_transversal">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#slider_transversal::groove:horizontal {
    border: 1px solid #999999;
    height: 8px;
    background: qlineargradient(
        spread:pad,
        x1:0.198, y1:0.903,
        x2:1, y2:0,
        stop:0.354167 rgba(213, 213, 213, 255),
        stop:0.661458 rgba(147, 147, 147, 255)
    );
    border-radius: 5px;
}

#slider_transversal::handle:horizontal {
    background: rgba(0, 0, 0, 189);
    border: 1px solid #5c5c5c;
    width: 18px;
    height: 18px;
    margin: -5px 0;
    border-radius: 9px;
}

#slider_transversal::sub-page:horizontal {
    background: rgba(147, 147, 147, 150);
    border-radius: 5px;
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_coronal">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#slider_coronal::groove:horizontal {
    border: 1px solid #999999;
    height: 8px;
    background: qlineargradient(
        spread:pad,
        x1:0.198, y1:0.903,
        x2:1, y2:0,
        stop:0.354167 rgba(213, 213, 213, 255),
        stop:0.661458 rgba(147, 147, 147, 255)
    );
    border-radius: 5px;
}

#slider_coronal::handle:horizontal {
    background: rgba(0, 0, 0, 189);
    border: 1px solid #5c5c5c;
    width: 18px;
    height: 18px;
    margin: -5px 0;
    border-radius: 9px;
}

#slider_coronal::sub-page:horizontal {
    background: rgba(147, 147, 147, 150);
    border-radius: 5px;
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_sagital">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>Corte Sagital</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>681</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color:rgba(0,0,0,100);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-30</y>
     <width>711</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/img/QS.jpg);
background-position: center;
background-repeat: no-repeat;
background-size:contain; /* o prueba &quot;contain&quot; si quieres que no recorte */
background-color: rgba(255, 255, 255, 0.8); /* m√°s opaco */
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Medium</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Transversal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Medium</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Coronal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>520</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Medium</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sagital</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>boton_volver</zorder>
  <zorder>label_transversal</zorder>
  <zorder>label_coronal</zorder>
  <zorder>slider_sagital</zorder>
  <zorder>slider_transversal</zorder>
  <zorder>slider_coronal</zorder>
  <zorder>label_sagital</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
